src/pkg/blob/models/blob.go

```diff
@@ -99,11 +103,37 @@ func (b *Blob) IsManifest() bool {
 // ProjectBlob alias ProjectBlob model
 type ProjectBlob = models.ProjectBlob
 
-// ListParams list params
-type ListParams struct {
-	ArtifactDigest  string    // list blobs which associated with the artifact
-	ArtifactDigests []string  // list blobs which associated with these artifacts
-	BlobDigests     []string  // list blobs which digest in the digests
-	ProjectID       int64     // list blobs which associated with the project
-	UpdateTime      time.Time // list blobs which update time less than updatetime
+// FilterByArtifactDigest returns orm.QuerySeter with artifact digest filter
+func (b *Blob) FilterByArtifactDigest(ctx context.Context, qs orm.QuerySeter, key string, value interface{}) orm.QuerySeter {
+	v, ok := value.(string)
+	if !ok {
+		return qs
+	}
+	sql := fmt.Sprintf("IN (SELECT digest_blob FROM artifact_blob WHERE digest_af IN (%s))", `'`+lib_orm.Escape(v)+`'`)
```

Comment:
`Escapse()` should only be used on fuzzy match

Reply:
will remove it.